packages/next/package.json

```diff
@@ -98,6 +99,7 @@
     "pnp-webpack-plugin": "1.6.4",
     "postcss": "8.2.13",
     "process": "0.11.10",
+    "prompts": "2.4.1",
```

Comment:
Introduced this to the repo (14.3KB gzipped) to show the prompt to the user when they run `next lint`.

If I shouldn't be including a dep for this, or if there's another smaller one I can use, let me know 🙏

Reply:
Its actually [200KB](https://packagephobia.com/result?p=prompts@2.4.1) since this is under `dependencies` so we could us `ncc` to bundle it down (maybe 100KB).

I'll defer to @timneutkens and @ijjk here

Reply:
Oof, you're right. Tried `nccing` it but it was showing some buggy behavior when compiled unfortunately.

Switched to using another library, [cli-select](https://www.npmjs.com/package/cli-select), that's a lot lighter ([28 KB](https://packagephobia.com/result?p=cli-select)). It doesn't have nearly as many downloads, but is light as it needs to be since I'm only using a single select prompt here

Also used `ncc` + a dynamic import. That brought down the total increase to `node_modules` from 350 to 184 KB, but happy to consider any other alternatives 🙏🏾